webviz_ert/plugins/_webviz_ert.py

```diff
@@ -1,15 +1,43 @@
-from typing import MutableMapping, Optional
+from typing import MutableMapping, Optional, Dict, Any
 import dash
 from webviz_config import WebvizPluginABC
 from webviz_ert.models import EnsembleModel
+import pathlib
+import json
+import tempfile
 
 
 class WebvizErtPluginABC(WebvizPluginABC):
     _ensembles: MutableMapping[str, "EnsembleModel"] = {}
+    _state: MutableMapping[str, Any] = {}
+    _state_file_name: str = "last_known_state.json"
+    _state_path: Optional[pathlib.Path] = None
 
     def __init__(self, app: dash.Dash, project_identifier: str):
         super().__init__()
+        if not project_identifier:
+            project_identifier = tempfile.NamedTemporaryFile().name
+            WebvizErtPluginABC._state = {}
+            WebvizErtPluginABC._state_path = None
+
         self.project_identifier: str = project_identifier
+        WebvizErtPluginABC._state = self.init_state(pathlib.Path(project_identifier))
```

Comment:
Do we load the file once for each plugin initialization? And put the result in the same global reference?

Reply:
No, the file is loaded only once the first time one of the plugins is initialized after that we are using the in-memory state. when loading things.